### DISCUSSION TITLE
Tools/visualizations proposed for incorporating in gEAR

### DISCUSSION THREAD

**Discussion by JPReceveur** [IGS/gEAR#328] [Ideas]
The following thread is a place to list and discuss the tools/ alternative visualizations user request for addition to the gEAR platform.

**Comment by JPReceveur** [comment]
Had a user request  for implementing [PAGODA](https://www.rna-seqblog.com/pagoda-characterizing-transcriptional-heterogeneity-through-pathway-and-gene-set-overdispersion-analysis/) tools for visualizing single-cell data in gEAR.

Even if we decide not to implement it, its an example of another online single cell tool to get some ideas from. It also looks like there is (or will be) functionality built in for joint analysis of datasets in pagoda2.

Github pages: https://github.com/kharchenkolab/pagoda2 (seems to be new version in development)

https://github.com/hms-dbmi/scde

<img width="1455" alt="Screen Shot 2022-05-20 at 8 11 37 AM" src="https://user-images.githubusercontent.com/28451557/169527493-f35a6aca-9549-4da9-ae5f-5302b347b3cf.png">

**Comment by songeric1107** [comment]
another option might be "[Cellenics]". the link is (https://scp.biomage.net/data-management).

here is the github page:
https://github.com/hms-dbmi-cellenics/ui

- Duo authorization.
- You can upload multiple samples at the same time
- Able to identify the doublet
- You can integrate multiple samples together
- You can compare cell types between groups or within group
- You can hide certain cell types
![Screen Shot 2022-05-27 at 8 41 26 AM](https://user-images.githubusercontent.com/6730079/170702080-06de2514-b3ac-438f-aa8a-f90a795f8815.png)
![Screen Shot 2022-05-27 at 8 40 59 AM](https://user-images.githubusercontent.com/6730079/170702089-9bf323aa-b93f-4723-93b5-d92cf12cf81f.png)

**Comment by adkinsrs** [comment]
Cellxgene (https://github.com/chanzuckerberg/cellxgene) has also been proposed in the past.
